app/storage.py

```
@@ -5,17 +5,26 @@
 from datetime import datetime
 from difflib import SequenceMatcher
 from typing import Awaitable, Literal, TypeVar
+from warnings import deprecated
```

Pylint (3.3.7)

[error] No name 'deprecated' in module 'warnings'

(E0611)